resources/views/checkout/pages/login.blade.php

```diff
@@ -0,0 +1,31 @@
+@extends('rapidez::layouts.app')
+
+@section('title', __('Checkout'))
+
+@section('robots', 'NOINDEX,NOFOLLOW')
+
+@section('content')
+    <div class="container">
+        <x-rapidez-ct::title-progress-bar :href="route('cart')" :$checkoutSteps :$currentStep :$currentStepKey>
```

Comment:
Can you check the `:href="route('cart')"` I think we don't need it here. As we don't use it in credentials or payment.

Reply:
Yes removed it!